docs/application/dotnet/guides/device/settings.md

```diff
@@ -26,14 +26,21 @@ To enable your application to use the system setting functionality:
     </privileges>
     ```
 
-2. To make your application visible in the Tizen Store only for devices that support the system setting features used in your application, add the corresponding feature keys to the `tizen-manifest.xml` file:
+2. To make your application visible in the Tizen Store only for devices that support the system setting features, add the following feature key to the `tizen-manifest.xml` file:
 
     ``` 
     <!--To use the WallpaperHomeScreen property and WallpaperHomeScreenChanged event-->
     <feature name="http://tizen.org/feature/systemsetting.home_screen"/>
     ```
 
-    The following table lists the feature keys required by the specific properties and events of the `Tizen.System.SystemSettings` class.
+    To use all the properties and events of [Tizen.System.SystemSettings](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.SystemSettings.html) class, add the following feature key to the `tizen-manifest.xml` file:
+    ``` 
+    <feature name="http://tizen.org/feature/systemsetting"/>
+    ```
+
+    Tizen Store can filter features based on the feature keys entered in the manifest file, and accordingly display apps to user's device.	
```

Comment:
Hello, Annie. I have a suggestion here. 
`Tizen Store can filter features based on the feature keys entered in the manifest file, and accordingly display apps to user's device.`
How about removing this sentence as it is guided on docs/application/native/tutorials/details/app-filtering-n.md and docs/application/web/tutorials/details/app-filtering-w.md? This sentence is about common information so that we don't need to add this sentence on every guide pages if we make a section for it.

Reply:
Fixed, removed this statement. We could link to application filtering topic (of dotnet) when it is created.